katello/katello-backup

```diff
@@ -63,6 +67,18 @@ def create_directories(directory)
   FileUtils.chmod_R 0770, directory
 end
 
+def confirm
+  puts "WARNING: This script will stop your services."
+  print "Are you sure(Y/N)? "
+  response = gets.chomp
+  if /[Y]/i.match(response)
+  else
+    puts "**** cancelled ****"
+    FileUtils.rm_rf @dir
+    exit(-1)
+  end
+end
```

Comment:
Highline has a simple [agree method](http://www.rubydoc.info/gems/highline/1.6.21/HighLine:agree) that could simplify this check

you can run do something like this
```
require 'highline' (at top)
response = HighLine.agree('WARNING: This script will stop your services. Do you want to proceed(y/n)?')
```